plugins/services/src/js/containers/tasks/TaskTable.js

```diff
@@ -322,6 +322,7 @@ class TaskTable extends React.Component {
 
     const dangerState = TaskStates[state].stateTypes.includes("failure");
     const activeState = TaskStates[state].stateTypes.includes("active");
+    const transitional = ["TASK_KILLING", "TASK_STARTING", "TASK_STAGING"].includes(state);
```

Comment:
Please run prettier, that line is a bit longer.

/jenkins/workspace/Frontend/public-dcos-ui-pulls/plugins/services/src/js/containers/tasks/TaskTable.js
  325:27  error  Follow `prettier` formatting (expected '\n' but found '"')  prettier/prettier

��� 1 problem (1 error, 0 warnings)